tendrl-api.spec

```diff
@@ -102,6 +103,10 @@ getent passwd %{app_user} > /dev/null || \
 %post httpd
 setsebool -P httpd_can_network_connect 1
 systemctl enable tendrl-api >/dev/null 2>&1 || :
+firewall-cmd --reload >/dev/null 2>&1 || :
+
+%postun
+firewall-cmd --reload >/dev/null 2>&1 || :
```

Comment:
Why reload command is called in two places?

Reply:
one is after installation and another one is after uninstall